src/components v2/Molecules/Components/Message/Message.vue

```diff
@@ -1,10 +1,11 @@
 <template>
-  <div ref="ph-message" class="photon-message" :class="[type]">
+  <div ref="ph-message" class="photon-message list" :class="[type]">
     <p-icon
       ref="messageIcon"
       :name="icon[type]"
       type="lg"
       class="icon"
+      color=""
```

Comment:
is the color required here

Reply:
so by setting it to empty it doesn't set its color to "inherit" which breaks the styles.